python/tvm/relay/to_onnx.py

```diff
@@ -0,0 +1,422 @@
+        input_name = 'shape{}'.format(node['output_names'][0])
+        node = onnx.helper.make_node(cls.__name__, [node['input_names'][0], input_name],
+                                     node['output_names'])
+        model_container.add_nodes([node])
+        input = onnx.helper.make_tensor_value_info(input_name,
+                                                   onnx.mapping.NP_TYPE_TO_TENSOR_TYPE[shape.dtype],
+                                                   shape=shape.shape)
+        model_container.add_inputs([input])
+        shape_tensor = numpy_helper.from_array(shape, input_name)
+        model_container.add_initializers([shape_tensor])
+
+
+class Conv(OpConverter):
+    """ Operator converter for Conv.
+    """
+    @classmethod
+    def convert_attributes(cls, attrs):
+        return {
+            'group': attrs.get_int("groups"),
+            'pads': attrs.get_int_tuple("padding") + attrs.get_int_tuple("padding"),
+            'strides': attrs.get_int_tuple("strides"),
+            'dilations': attrs.get_int_tuple("dilation"),
+            'kernel_shape': attrs.get_int_tuple("kernel_size"),
+            }
+
+
+class MaxPool(OpConverter):
+    """ Operator converter for MaxPool.
+    """
+    @classmethod
+    def convert_attributes(cls, attrs):
+        return {
+            'pads': attrs.get_int_tuple("padding") + attrs.get_int_tuple("padding"),
+            'strides': attrs.get_int_tuple("strides"),
+            'kernel_shape': attrs.get_int_tuple("pool_size"),
+            }
+
+
+class Transpose(OpConverter):
+    """ Operator converter for Transpose.
+    """
+    @classmethod
+    def convert_attributes(cls, attrs):
+        return {'perm': attrs.get_int_tuple("axes")} if attrs["axes"] else {}
+
+
+class MatMul(OpConverter):
+    """ Operator converter for MatMul.
+    """
+    @classmethod
+    def convert(cls, node, model_container, node_list):
+        output_name = 'inter{}'.format(node['output_names'][0])
+        transpose_node = onnx.helper.make_node(Transpose.__name__,
+                                               [node['input_names'][1]],
+                                               [output_name],
+                                               **{'perm': (1, 0)})
+        model_container.add_nodes([transpose_node])
+
+        inputs = [node['input_names'][0], output_name]
+        matmul_node = onnx.helper.make_node(cls.__name__, inputs, node['output_names'])
+        model_container.add_nodes([matmul_node])
+
+
+class Flatten(OpConverter):
+    """ Operator converter for Flatten.
+    """
+    @classmethod
+    def convert_attributes(cls, attrs):
+        return {
+            'axis': 1,
+            }
+
+
+class BatchNormalization(OpConverter):
+    """ Operator converter for BatchNormalization.
+    """
+    @classmethod
+    def convert_attributes(cls, attrs):
+        return {
+            'epsilon': float(attrs.get_str('epsilon')),
+            'momentum': float('0.8999999761581421'),
```

Comment:
Why this number?

Reply:
Removed. It was added for debugging. Got pushed by mistake.